apps/condo/domains/user/schema/RegisterNewUserService.test.js

```diff
@@ -45,9 +45,14 @@ describe('RegisterNewUserService', () => {
 
         expect(data).toEqual({ 'user': null })
         expect(errors).toMatchObject([{
-            'message': '[format:phone] invalid format',
-            'name': 'GraphQLError',
-            'path': ['user'],
+            message: 'Wrong format of provided phone number',
+            name: 'GraphQLError',
+            path: ['user'],
```

Comment:
should be `['user','phone']`

Reply:
No, it's an Apollo's part, where it points a node of GraphQL request, where error was occurred. We have `['user']` here, because we have declared our mutation as `user: registerNewUser`.